### DISCUSSION TITLE
JDA needs voice channel status update

### DISCUSSION THREAD

**Discussion by lajczik** [discord-jda/JDA#2562] [Feedback & Suggestions]
Discord added a new permission "**Set Voice Channel Status**" which is missing in the **Permissions** enum.
![image](https://github.com/discord-jda/JDA/assets/55200929/c4f363a4-c7dd-4551-ab2d-abf8641b5314)

In JDA I miss changing the voice channel status, like in screenshot:
![image](https://i.imgur.com/NZYFCI8.png)

**Comment by MinnDevelopment** [comment]
Discord has yet to document it, but there is an open PR already #2532
